parachain/primitives/router/src/outbound/mod.rs

```diff
@@ -283,17 +288,17 @@ impl<'a, Call> XcmConverter<'a, Call> {
 			} = asset_location
 			{
 				if gateway_network.is_some() && gateway_network != &Some(*self.ethereum_network) {
-					return Err(AssetResolutionFailed)
+					return Err(AssetResolutionFailed);
 				}
 				if gateway_address != self.gateway_address {
-					return Err(AssetResolutionFailed)
+					return Err(AssetResolutionFailed);
 				}
 				if token_network.is_some() && token_network != &Some(*self.ethereum_network) {
-					return Err(AssetResolutionFailed)
+					return Err(AssetResolutionFailed);
 				}
 				(token_address.into(), *amount)
 			} else {
-				return Err(AssetResolutionFailed)
+				return Err(AssetResolutionFailed);
```

Comment:
Ditto, your formatter may be misconfigured.

Reply:
Fixed in https://github.com/Snowfork/snowbridge/pull/972/commits/cc9b1fc2aa6f50764287be80db7eb4e8666d4fe7.